kazoo/tests/test_retry.py

```diff
@@ -1,89 +1,107 @@
-import unittest
-
 import pytest
 
 
-class TestRetrySleeper(unittest.TestCase):
```

Comment:
Can I ask why not using a class anymore? It seems to make the `_try_counts` handling a little more "simple" (ie. without the use of `global` + doing its reinit in the `setup()` method) but I might be missing something.

Reply:
I cleaned this up.
I removed the class to get rid of the unittest model.
I think I cleaned this up enough now that it is actually better.

Reply:
FWIW, I'm a fan of getting rid of the Unittest subclassing model whenever possibel... I personally find it clunky compared to the pytest syntax.